locksmith/__tests__/signatureValidationMiddleware.test.ts

```diff
@@ -1,7 +1,8 @@
-var httpMocks = require('node-mocks-http')
-var signatureValidationMiddleware = require('../src/signatureValidationMiddleware')
+import httpMocks = require('node-mocks-http')
+import express = require('express')
+import signatureValidationMiddleware = require('../src/signatureValidationMiddleware')
 
-let request, response
+let request: httpMocks.MockRequest<express.Request>, response: httpMocks.MockResponse<express.Response>
```

Comment:
Generics! Was slightly difficult to figure out what the correct type was here; TS didn't infer it so it was  `any`